src/index.js

```diff
@@ -1,3 +1,3 @@
 import WordCloud from './WordCloud';
 
-module.exports = WordCloud;
```

Comment:
We should not change this, if we change this, we must use `default` to require it:

```js
const WordCloud  = require('react-d3-cloud').default;
```